scripts/ci/ensure-angular-dependencies-compatibility.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * This script is in charge of making sure that the different modules
+ * that compose the AngularJS framework have compatible version.
+ *
+ * See: https://github.com/resin-io/etcher/pull/1168
+ */
+
+'use strict';
+
+const _ = require('lodash');
+const EXIT_CODES = require('../../lib/shared/exit-codes');
+const packageJSON = require('../../package.json');
+
+const dependencies = [
+  {
+    name: 'angular',
+    value: packageJSON.dependencies.angular
+  },
+  {
+    name: 'angular-mocks',
+    value: packageJSON.devDependencies['angular-mocks']
+  }
+];
+
+_.each(dependencies, (dependency) => {
+  if (!/^\d+\.\d+\.\d+/.test(dependency.value)) {
+    console.error(`Dependency: ${dependency.name} should be locked`);
```

Comment:
As long as we're checking that they're the same version, does it matter if they're locked or not?